Remotion/Core/UnitTests/Reflection/TypeDiscovery/AssemblyFinding/AssemblyFinderIntegrationTest.cs

```diff
@@ -30,7 +30,9 @@ namespace Remotion.UnitTests.Reflection.TypeDiscovery.AssemblyFinding
 {
   [TestFixture]
   [Serializable]
-  [Ignore ("TODO RM-7808: Integrate the RoslynCodeDomProvider and renable the AssemblyCompiler tests")]
+  #if !NETFRAMEWORK
```

Comment:
The `NETFRAMEWORK` pre-processor directive needs to be added to all the projects. Both in the `DEBUG` and the `RELEASE` configuration using the `DefineConstants` property